### PR TITLE
Move more internal packages to be internal

### PR DESCRIPTION
**Pull request by iHiD**

### CODE REVIEW

**Review by ErikSchierboom — APPROVED** [review]

**Review by dem4ron — APPROVED** [review]

**Comment by iHiD** [comment]
@dem4ron Could you see why this isn't green please, and add commits to fix it. Specifically, why has this PR caused these tests to fail?

File: app/javascript/packs/internal.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import 'easymde/dist/easymde.min.css'
 
 import React, { lazy, Suspense } from 'react'
+import currency from 'currency.js'
```
Comment (iHiD) [inline app/javascript/packs/internal.tsx:5]: @dem4ron can we remove this from application.tsx now?

Comment (dem4ron) [reply app/javascript/packs/internal.tsx:5]: Done

**Comment by iHiD** [comment]
@dem4ron I think this is causing https://forum.exercism.org/t/unable-to-subscribe-to-premium/6560

Could you work out why the tests didn't pick this up pls?

**Pull request merged**